libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager+Internal.h

```diff
@@ -65,6 +65,11 @@
  */
 @property (nonatomic, strong, nullable) SFSDKAuthErrorManager *errorManager;
 
+/**
+ Whether the app is configured to require certificate-based authentication. (RequireCertAuth)
```

Comment:
Tweak this comment to refer to browser-based authentication instead of certificate-based authentication.